Create a chapter on Book Dash (for the community handbook) (#1262)
alan-turing-institute/the-turing-way: book/website/community-handbook/bookdash/bookdash-events.md

It is an opportunity for the organising team and the selected attendees to get to know each other and discuss any concerns that they may have prior to the book dash.

Several participants often come to these calls with some plans and topics they would like to explore for their contributions in _The Turing Way_.
These topics can be something that is either missing in the existing chapters or need to be updated in the project.
We also discuss _The Turing Way_ as a project in general and what the core team is working on to support the development of its guide on reproducibility, ethics, project design, collaboration, communication and a meta-book on the project itself.
Most importantly, the core team goes through the contribution guideline and explain how attendees can effectively use their time at the book dash.

The template for shared-notes can be accessed in the community handbook {ref}`here<ch-bookdash-precall-template>`.
You can read notes from the calls from the most recent book dash [here](https://hackmd.io/@malvikasharan/bookdash).

> ## Day-1: brainstorming and dinner

Reviewer: Arguably a personal preference, but I would put the schedule overview first and then the expanded descriptions

Author: I will move the schedule above. It does make it easier to read that way.

Author: Moved the agenda to the top: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/642fd193ae64a462e37ccd51a282c7112d703473

The invited participants join the core team for an introduction session a day before the book dash event.
This is a chance to everyone to meet the participants and helpers informally, learn about each other and connect with specific members who plan to work on related topic during the book dash.

![](./figures/book-dash-feb20-intro1.jpg)
*Three of our participants, Heidi, Esther and Adina, exchanging local sweets and snacks from their home country at the beginning of the event (picture by Kirstie Whitaker)*

> 

Author: Added in the section below.

In a dedicated hour, the core team members provide an overview of _The Turing way_ project and resources that are available for the participants for their book dash contributions.

> The next hours are dedicated for brainstorming in small groups, organising thoughts and ideas for their contributions the next day, share their plans with each other at the end of the session and identify their collaborators based on the similarity of their topics of interest.

Reviewer: "The next hours are dedicated to brainstorming in small groups, organising thoughts and ideas for contributions the next day, sharing plans with other participants at the end of the session and identifying collaborators based on the similarity of their interests." 

OR 

" The next hours are dedicated **to the participants** brainstorming in small groups, organising their thoughts and ideas for contributions the next day, sharing their plans with other participants at the end of the session and identifying their collaborators based on the similarity of their interests."

> The next hours are dedicated for brainstorming in small groups, organising thoughts and ideas for their contributions the next day, share their plans with each other at the end of the session and identify their collaborators based on the similarity of their topics of interest.

Author: ```suggestion
The next hours are dedicated to the participants brainstorming in small groups, organising their thoughts and ideas for contributions the next day, sharing their plans with other participants at the end of the session and identifying their collaborators based on the similarity of their interests.

For participants who are unfamiliar with GitHub or would like a refresher, the core team members and volunteers guide them through the process of making their first pull request.
```

The day ends with a group dinner hosted by the core team to allow participants to relax after an intense discussion session and enjoy meals together in an informal setting.
In between the meals, participants are invited to give lightning talks where they can share their personal stories and talk about a project that they’re either working on or are excited about.

> 

Reviewer: Suggest to add: 

"Lightning talks are short (normally 2 minutes long) and are given without slides."

> 

Author: ```suggestion
Lightning talks are short (normally 2 minutes long) and are given without slides.
```

### Agenda for the Day-1

> 

Reviewer: Does timezone matter in the agenda for the purposes of the chapter?

Author: It should not. matter really.  I can take that out.

Author: Removed BST: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/6f396e3c3e8085674775a1c2cffb2bcb7b863d17

```
14:00-15:00 BST: Arrive at the book dash venue for informal networking
15:00-16:00 BST: Intro to _The Turing Way_ and book dash
17:00-17:30 BST: Brainstorming in groups
17:30-18:00 BST: Report out
18:30 BST: Arrive at the dinner venue
19:00 BST: Dinner begins
20:30-21:00 BST: Lightning talks*
21:00-22:00 BST: Dinner continues with networking
```

## Day-2: coworking day

> 

Reviewer: Again, would suggest that the agenda goes up top followed by the descriptions.

The next day of the book dash is a collaboration and coworking day.
This day is distributed into 4 sessions of 1 hour 15 minutes each, separated by 15 minutes reflection followed by a break.

The first session facilitates discussions by bringing all the attendees in small groups with their collaborators based on their discussion from the previous day.
The collaborating groups start working on their road map and plans for their contributions as chapters, subchapters or other types of contents.
Participants are also encouraged to come up with the main sections of their contributions and collect their thoughts in bullet points on a GitHub issue.
During the second session, participants work in self-organised groups or individually with the support of organisers and helpers.
This session ends with a quick report out from each group followed by a lunch break.

> During the third and fourth session, participants come back to continue working uninterrupted on their contribution ideas and take break as they need.

Reviewer: "take breaks"

> During the third and fourth session, participants come back to continue working uninterrupted on their contribution ideas and take break as they need.

Author: ```suggestion
During the third and fourth session, participants come back to continue working uninterrupted on their contribution ideas and take breaks as they need.
```

At the end of this session, participants gather all their contributions  on _The Turing Way_ GitHub repository (issues, pull requests and notes) and add that in the shared notes before taking a short break.

The last session is the final report out and celebration of each other's work.
The book dash closes with anonymous feedback on the {ref}`Pluses and Delta<ch-bookdash-feedback-template>` notes.

### Agenda for Day-2

```
8:30 BST: Arrive at the dashing venue
8:45 BST: Intro to the second day of the book dash
9:00 BST: Coworking starts (session-1)
10:15 BST: Report
11:00 BST: Coworking continues (session-2)
12:15 BST: Report out
12:30 BST: Lunch break
13:30 - 16:30 BST: Coworking continues (session-3 and 4)
16:30 BST: Group share outs and celebrations
17:00 BST: Event closing and feedback
```

## Creating an environment for self-paced contributions

> We acknowledge that everyone prefer different setting to work effectively and attentively.

Author: ```suggestion
We acknowledge that everyone prefers different settings to work effectively and attentively.
```

> Therefore we create environment for self-paced contributions to _The Turing Way_ project.

Reviewer: "we create an environment"

> Therefore we create environment for self-paced contributions to _The Turing Way_ project.

Author: ```suggestion
Therefore we create an environment for self-paced contributions to _The Turing Way_ project.
```

> Therefore we create environment for self-paced contributions to _The Turing Way_ project.

Author: ```suggestion
Therefore we create an environment for self-paced contributions to _The Turing Way_ project.
```

> Some recommendation to achieve that is by managing space by:

Reviewer: Suggest to change to either: 
"Some recommendations to achieve this by managing the space at the book dash by:" 
OR 
"Some recommendations to achieve this by managing space are:"

> Some recommendation to achieve that is by managing space by:

Author: ```suggestion
Some recommendations to achieve this by managing the space at the book dash by:
```

Participants can sign up on this HackMD to attend the upcoming calls: https://hackmd.io/@KirstieJane/CollabCafe.
Anyone who would like to host these calls at other times, please get in touch with the team members.
See more details in the chapter {ref}`Coworking Call<ch-coworking-calls>`

### Reviewing an open pull requests

Our contributors can help us by reviewing one of the several open pull requests based on their interest.
They can join discussions on these pull requests or help in exploring resources that currently exists in the book.

### Representing _The Turing Way_ in your community/conference

If our attendees would like to represent our community, they can connect with _The Turing Way_ team for support and assistance in drafting an abstract and help them prepare to deliver a short workshop or presentation.

### Connecting through social media

Everyone is welcome to join our Gitter channel for informal discussions: .
They can also sign up to receive our monthly newsletter: https://tinyletter.com/TuringWay/.

> We are on Twitter as [@turingway](), where we send regular updates.

Reviewer: Missing Twitter account link?

Author: Added now.

> We are on Twitter as [@turingway](), where we send regular updates.

Author: ```suggestion
We are on Twitter as [@turingway](https://twitter.com/turingway), where we send regular updates.
```

If you have more ideas or questions about the project, the book dash event or something else related to our community, please feel free to reach out to the team members (theturingway@gmail.com).

## Report and additional materials

> - [Report from February 2020](https://hackmd.io/@malvikasharan/rJSSJurEL) (currently on [GitHub PR](https://github.com/alan-turing-institute/the-turing-way/pull/928))

Author: ```suggestion
- [Report from February 2020](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-feb20-report.md)
```

- [Report from London 2019](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-ldn-report.md)

> - [Report from Manchester 2019](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-mcr-report.md)

Author: ```suggestion
- [Report from Manchester 2019](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-mcr-report.md)
- Additional [resources and organisation files](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash)
```
